src/rpc/blockchain.cpp

```diff
@@ -2143,7 +2147,8 @@ UniValue scantxoutset(const JSONRPCRequest& request)
             tip = ::ChainActive().Tip();
             CHECK_NONFATAL(tip);
         }
-        bool res = FindScriptPubKey(g_scan_progress, g_should_abort_scan, count, pcursor.get(), needles, coins);
+        NodeContext& node = EnsureNodeContext(request.context);
+        bool res = FindScriptPubKey(g_scan_progress, g_should_abort_scan, count, pcursor.get(), needles, coins, node);
```

Comment:
nit: why not pass in the `rpc_interruption_point` like below  for `GetUTXOStats`?

```suggestion
        bool res = FindScriptPubKey(g_scan_progress, g_should_abort_scan, count, pcursor.get(), needles, coins, node.rpc_interruption_point);
```

Reply:
[Updated](https://github.com/bitcoin/bitcoin/pull/19323#issuecomment-646980904).